hushline/model.py

```diff
@@ -75,6 +77,40 @@ class ExtraField:
     is_verified: Optional[bool]
 
 
+@enum.unique
+class StripeInvoiceStatusEnum(enum.Enum):
+    DRAFT = "draft"
+    OPEN = "open"
+    PAID = "paid"
+    UNCOLLECTIBLE = "uncollectible"
+    VOID = "void"
+
+
+@enum.unique
+class StripeSubscriptionStatusEnum(enum.Enum):
+    INCOMPLETE = "incomplete"
+    INCOMPLETE_EXPIRED = "incomplete_expired"
+    TRIALING = "trialing"
+    ACTIVE = "active"
+    PAST_DUE = "past_due"
+    CANCELED = "canceled"
+    UNPAID = "unpaid"
+    PAUSED = "paused"
+
+
+@enum.unique
+class StripeEventStatusEnum(enum.Enum):
+    PENDING = "pending"
+    IN_PROGRESS = "in_progress"
+    ERROR = "error"
+    FINISHED = "finished"
+
+
+# Premium tiers
+FREE_TIER = 1
```

Comment:
We might want to move these into the `Tier` model and stick to that pattern generally, where a constant related to a class is always part of that class.

Reply:
Fixed in 3757b5ec6137201111e00ed8335b73f8e5388721